lib/pages/home/modals/post_comment/post-commenter-expanded.dart

```diff
@@ -117,20 +117,24 @@ class OBPostCommenterExpandedModalState extends State<OBPostCommenterExpandedMod
 
   void _onWantsToSaveComment() async {
     PostComment comment;
-    if (widget.postComment != null) {
-      comment = await _userService.editPostComment(
-          post: widget.post,
-          postComment: widget.postComment,
-          text: _textController.text);
-    } else {
-      comment = await _userService.commentPost(
-          post: widget.post,
-          text: _textController.text);
-    }
-
-    if (comment != null) {
-      // Remove modal
-      Navigator.pop(context, comment);
+    try {
```

Comment:
instead of hiding the edit and post buttons on closed posts, i chose to show them but relay proper errors from backend so people learn the behaviour of the app.

Reply:
No man.. It's an odd behaviour in an a single place. Sets the wrong expectations over the UI. As in expecting always to see errors where not even possible to perform X action, instead of simply not having it.

Reply:
cool makes sense